downstream/modules/platform/proc-controller-create-credential-type.adoc

```diff
@@ -0,0 +1,214 @@
+[id="proc-controller-create-credential-type"]
+
+= Create a New Credential Type
+
+To create a new credential type:
+
+.Procedure
+. In the *Credential Types* view, click btn:[Add].
++
+image:credential-types-create-new.png[Create new credential type]
+
+. Enter the appropriate details in the *Name* and *Description* field.
++
+[NOTE]
+====
+When creating a new credential type, do not use reserved variable names that start with `ANSIBLE_` for the *INPUT* and *INJECTOR* names and IDs, as they are invalid for custom credential types.
+====
+
+. In the *Input Configuration* field, specify an input schema which defines a set of ordered fields for that type. 
+The format can be in YAML or JSON, as shown:
++
+*YAML*
++
+[literal, options="nowrap" subs="+attributes"]
+----
+fields:
+  - type: string
+    id: username
+    label: Username
+  - type: string
+    id: password
+    label: Password
+    secret: true
+required:
+  - username
+  - password 
+----
++
+View more YAML examples at the link:https://yaml.org/spec/1.2.2/[YAML page]
++
+*JSON*
++
+[literal, options="nowrap" subs="+attributes"]
+----
+{
+"fields": [
+  {
+  "type": "string",
+  "id": "username",
+  "label": "Username"
+  },
+  {
+  "secret": true,
+  "type": "string",
+  "id": "password",
+  "label": "Password"
+   }
+  ],
+ "required": ["username", "password"]
+}
+----
++
+View more JSON examples at link:https://www.json.org/json-en.html[The JSON website]
++
+The configuration in JSON format below show each field and how they are used:
```

Comment:
Suggested re-write:

The following configuration in JSON format shows each field and how they are used: